main.py

```diff
@@ -0,0 +1,10 @@
+import asyncio
+
+from aiohttp import web
+
+from app import init_app
+
+if __name__ == '__main__':
+    loop = asyncio.get_event_loop()
+    app = loop.run_until_complete(init_app())
+    web.run_app(app)
```

Comment:
```suggestion
    web.run_app(init_app())
```
https://docs.aiohttp.org/en/stable/web_advanced.html#passing-a-coroutine-into-run-app-and-gunicorn

Also, consider renaming `init_app` to `make_app`, `create_app` or even `app_factory`. I think this would be better suited to this situation.

Reply:
Renamed to `app_factory`, changed to passing coroutine into `web.run_app`